src/components/global/navigation/side.tsx

```diff
@@ -21,7 +21,7 @@ const SideNav = ({ links }: props) => {
           <Link
             key={index}
             href={link}
-            className="!w-full hover:bg-starlight-yellow"
+            className="block !w-full hover:bg-starlight-yellow"
```

Comment:
Forced all link objects to act as block elements so that they newline correctly. Not sure if there's a better way to have it newline

Reply:
`flex-col` on the container div should work? try that